tests/lib/rules/no-extra-parens.js

```diff
@@ -1122,12 +1153,10 @@ ruleTester.run("no-extra-parens", rule, {
                 }
             ]
         },
-
-        // ["all", { enforceForArrowConditionals: false }]
         {
             code: "var a = (b) => ((1 ? 2 : 3))",
             output: "var a = (b) => (1 ? 2 : 3)",
-            options: ["all", { enforceForArrowConditionals: false }],
+            options: ["all", { enforceForArrowConditionals: true }],
```

Comment:
Just to double check I'm understanding all these test changes correctly, this is the only test covering the code change in this PR, correct? All the others look to be covering existing behavior.

Reply:
@kaicataldo 
Oh, it seems needs more test cases, I'll do that. Thanks:)

Reply:
Oh no, sorry, I just wanted to make sure I understood why all the test cases were added!